### PR TITLE
fix: Fix missing response types in Swagger API endpoints

### PR DESCRIPTION
**Pull request by lan-yonghui**
Refs #7607

### CODE REVIEW

File: backend/app/api/v1/website.go [hunk 1]
```diff
@@ -785,7 +785,7 @@ func (b *BaseApi) GetDirConfig(c *gin.Context) {
 // @Summary Get default html
 // @Accept json
 // @Param type path string true "type"
-// @Success 200 {object} response.FileInfo
+// @Success 200 {object} response.WebsiteHtmlRes
 // @Security ApiKeyAuth
 // @Security Timestamp
 // @Router /websites/default/html/{type} [get]
```
Comment (wanghe-fit2cloud) [inline backend/app/api/v1/website.go:791]: The code changes look generally appropriate and follow standard API documentation conventions, but there are some slight inconsistencies:

1. **Status Code Mismatch**: The `@Success` annotations for methods that return arrays or objects (like `List website names`, `Get site permissions`, etc.) should use `{array}` instead of just `string`. For instance:
   ```go
   // @Success 200 {array} response.WebsiteOption
   ```

2. **Response Objects**: Some responses might not explicitly define their response structure using types like `response.NginxRewriteRes`, `response.WebsiteProxyConfig` etc. While it's generally okay to omit defining a specific type if it's straightforward and clear what data is expected, it can be beneficial to define custom structs to ensure consistency and clarity.

3. **File Info Object**: Replace `{object}` with `{object} response.FileInfo` in the `/websites/default/html/{type}` method since `FileInfo` represents file information which needs a more precise response definition.

These adjustments will improve readability and maintainability of your API endpoint definitions.

File: backend/app/api/v1/file.go [hunk 2]
```diff
@@ -786,7 +786,7 @@ func (b *BaseApi) Keys(c *gin.Context) {
 // @Tags File
 // @Summary Read file by Line
 // @Param request body request.FileReadByLineReq true "request"
-// @Success 200
+// @Success 200 {object} response.FileLineContent
 // @Security ApiKeyAuth
 // @Security Timestamp
 // @Router /files/read [post]
```
Comment (wanghe-fit2cloud) [inline backend/app/api/v1/file.go:792]: 1. The first line of the `ListFiles` function should be changed to use a struct literal instead of casting.
```go
files, err := fileService.GetFiles(req)
```

2.`helper.ErrorWithDetail(c, constant.CodeErrInternalServer, constant.ErrTypeInternalServer, err)` can be simplified to `helper.FailWithDetail(c, "internal server error", err)`
```go

err = fileService.GetFileList(req)
if err != nil {
        helper.FailWithDetail(c, "failed to get file list", err)
        return
}

helper.SuccessWithData(c, fileList)

```

3. The change from using `constant.ErrTypeInternalServer` to just stating `"internal server error"` is more explicit and cleaner.

4. In the `Keys` function, if there are multiple keys that need to be returned, it might be better to have those values stored in an array or map before passing them to `SuccessWithData`.

In summary, these changes streamline some boilerplate code such as logging errors or formatting responses and make them easier to read and maintain.

File: backend/app/api/v1/host_tool.go [hunk 3]
```diff
@@ -161,7 +161,7 @@ func (b *BaseApi) GetProcess(c *gin.Context) {
 // @Summary Get Supervisor process config
 // @Accept json
 // @Param request body request.SupervisorProcessFileReq true "request"
-// @Success 200
+// @Success 200 {string} content
 // @Security ApiKeyAuth
 // @Security Timestamp
 // @Router /host/tool/supervisor/process/file [post]
```
Comment (wanghe-fit2cloud) [inline backend/app/api/v1/host_tool.go:167]: There are no irregularities or significant issues with this Go/Gin API code snippet based on the provided knowledge cutoff of September 1, 2021. However, here are some minor observations:

1. **Comment Update**: The comment block for `/host/tool/supervisor/process` should reflect that it serves "Supervisor process config" rather than just being `true`.

```go
// @Summary Get Supervisor process config
// @Desc        Retrieve the configuration details for Supervisor processes.
```

This change ensures clarity about what this endpoint actually accomplishes.

2. **Response Type for `/host/tool/log`**:
   - Currently, `/host/tool/log` uses `"logContent"` in its `@Success` tag, which is not consistent. It might be more appropriate to return the actual log content, not just a placeholder string. For example:
     ```json
     ["error", "warning", "info"]
     ```
   
3. **Consistency Across Endpoints**: Ensure that all endpoints use uniform JSON responses unless they have specific data formats. This can make them easier to consume across different client applications.

4. **Logging**: Ensure proper logging throughout the endpoints if required for debugging or monitoring purposes.

5. **Error Handling**: Consider implementing better error handling practices, especially around invalid input data or unexpected errors during API operations.

These points, while not breaking any fundamental aspects of the code, could improve readability, consistency, and maintainability.

**Review by wanghe-fit2cloud — COMMENTED** [review]
/lgtm
